skymp5-client/src/front/components/movementGet.ts

```diff
@@ -10,19 +10,19 @@ export const getMovement = (refr: ObjectReference): Movement => {
   const runMode = ac ? getRunMode(ac) : "Running";
 
   let healthPercentage = ac && ac.getActorValuePercentage("health");
+  let staminaPercentage = ac && ac.getActorValuePercentage("stamina");
+  let magickaPercentage = ac && ac.getActorValuePercentage("magicka");
   if (ac && ac.isDead()) {
     healthPercentage = 0;
+    staminaPercentage = 0;
```

Comment:
Why do dead actors have zero stamina? Btw the task isn't to modify movement, please remove these diffs.